spec/furniture/marketplace/delivery_areas_controller_request_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe Marketplace::DeliveryAreasController, type: :request do
+  subject(:result) do
+    perform_request
+    test_response
+  end
+
+  before { sign_in(space, person) }
+
+  let(:space) { marketplace.space }
+  let(:marketplace) { create(:marketplace) }
+  let(:person) { create(:membership, space: space).member }
+
+  describe "#new" do
+    let(:perform_request) { get polymorphic_path(marketplace.location(:new, child: :delivery_area)) }
+
+    it { is_expected.to be_ok }
```

Comment:
It may be nice to leverage `assert_select` to make assertions about what the page will display, but I also think we can skip on that for now.